doc/ranger.pod

```diff
@@ -1840,6 +1841,19 @@ I<highlight> will pick up command line options specified in this variable. A
 C<--style=> option specified here will override C<HIGHLIGHT_STYLE>. Similarly,
 C<--replace-tabs=> will override C<HIGHLIGHT_TABWIDTH>.
 
+=item OPENSCAD_COLORSCHEME
+
+Specifies the colorscheme used by I<openscad> while previewing 3d models. The
+available colorschemes are. Cornfield, Metallic, Sunset, Starnight, BeforeDawn,
```

Comment:
Please list instructions for how to find out the options rather than listing the options. That way there's nothing to get out of date.

Reply:
I agree, funny thing is that the man page of openscad has an outdated list. Not our problem though.

Reply:
`%s/3d/3D` done

Reply:
Let's not make it our problem then. Say, "check the openscad manpage" or "check this or that wiki page," whatever you did to find them out, rather than listing things that will get outdated.